packages/compass-aggregations/src/components/input-preview/input-preview.module.less

```diff
@@ -13,37 +13,28 @@
     align-items: flex-start;
     overflow-x: scroll;
     margin: 10px;
+    padding: 10px;
+  }
 
-    :global {
-      .document {
-        background-color: @pw;
-        margin: 10px;
-        padding: 0px 20px;
-        border: 1px solid @gray6;
-        border-radius: 4px;
-        box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.1);
-        width: 350px;
-        height: 150px;
-        overflow: scroll;
-        flex-shrink: 0;
-
-        &-elements {
-          padding: 0px;
-          margin-bottom: 0px;
-        }
-      }
+  &-document-card {
+    flex: none;
+    border: 1px solid @gray6;
+    border-radius: 4px;
+    box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.1);
+    width: 350px;
+    height: 150px;
+    overflow: scroll;
```

Comment:
Comparing this branch to current GA:
<img width="700" alt="Screen Shot 2022-03-30 at 8 58 26 AM" src="https://user-images.githubusercontent.com/1791149/160840020-fee88944-6712-41c9-a3b5-f35f46ab7587.png">

I like the top margin reduction, how would you feel about a bit less left padding as well? Mostly thinking the more of the document we can show to folks in the preview the better.